grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesReader.groovy

```diff
@@ -33,19 +33,37 @@ import javax.jcr.Node as JcrNode
 @SuppressWarnings("GrMethodMayBeStatic")
 class JcrNodesReader implements ItemReader<JcrNode> {
 
+    private String excludePaths
+    private Collection<String> excludePathList
+
+    public void setExcludePaths(String excludePaths) {
+        if (excludePaths != null && !excludePaths.isEmpty()) {
+            this.excludePathList = excludePaths.split("<sp>") as Collection<String>
```

Comment:
Per [the JCR spec for path naming](http://www.day.com/specs/jcr/2.0/3_Repository_Model.html#3.2.4%20Naming%20Restrictions), "&lt;sp>" is perfectly legal (if unlikely) to be part of a path. You might want to use colon instead, since it's disallowed.